api/task_check_run.go

```diff
@@ -37,6 +37,19 @@ const (
 	TaskCheckStatusError TaskCheckStatus = "ERROR"
 )
 
+// Level returns the status level, ERROR=0, WARN=1, SUCCESS=2.
+func (t TaskCheckStatus) Level() int {
```

Comment:
I'd like to expose fewer magic numbers. Is it possible to expose Less() for comparing Success/Warn/Error cases?